### PR TITLE
Adding manage users section for profile page

### PR DESCRIPTION
**Pull request by maher-suleyman**
PTAL

### CODE REVIEW

File: src/containers/profile/ProfileSections/ManageUsers/ManageUsersSections/ProfileBox.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { updateProfile } from "../../../../../actions/authActions";
+import { Dropdown, Button, Row, Col, ListGroup, Image } from "react-bootstrap";
+import "./ProfileBox.css";
+import anonymousImage from "../../../../../images/anonymousImage.png";
+
+const ProfileBox = (props) => {
+  const { profile, auth } = props;
+  const photoSRC = profile.imageURL ? profile.imageURL : anonymousImage;
```
Comment (louisrli) [inline src/containers/profile/ProfileSections/ManageUsers/ManageUsersSections/ProfileBox.jsx]: photoSrc

File: src/containers/profile/ProfileSections/ManageUsers/ManageUsersSections/ProfileBox.jsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                </div>
+              </ListGroup.Item>
+            </ListGroup>
+          </Link>
+        </Col>
+        <Col
+          xs={4}
+          className="d-flex flex-column pt-2 pb-2 justify-content-around"
+        >
+          {!profile.isBlocked ? (
```
Comment (louisrli) [inline src/containers/profile/ProfileSections/ManageUsers/ManageUsersSections/ProfileBox.jsx]: too many conditions here, hard to read. move this whole conditional logic out into a variable above the `return` and use a normal if condition.

**Review by louisrli — APPROVED** [review]

**Review by ttugba — APPROVED** [review]
Good job!

**Comment by maher-suleyman** [comment]
> Good job!

thank you

**Comment by maher-suleyman** [comment]
PTAL

**Pull request merged**